### DISCUSSION TITLE
如何在目录增加 攻读学位期间获得的奖项的章节

### DISCUSSION THREAD

**Discussion by sdfgh468** [sjtug/SJTUThesis#975] [提问]
目前模板里只有“攻读学位期间学术论文和科研成果目录”和“攻读学位期间参与的项目”，如何在目录中新增一个章节“攻读学位期间获得的奖项”，试过直接在正文achievements.atex里加入（如下），但是目录里还是没有这个题目，求解答！

![1695363603394](https://github.com/sjtug/SJTUThesis/assets/22966522/c392af55-51a8-45b8-958a-facc6ee33f49)

**Comment by LogCreative** [comment]
```latex
\begin{achievements}[攻读学位期间获得的奖项]
   % ...
\end{achievements}
```

**Reply by sdfgh468** [reply]
感谢！试了你的方法，目录上确实可以显示“攻读学位期间获得的奖项”，但论文里的大标题还是“攻读学位期间学术论文和科研成果目录”，想改为“攻读学位期间获得的奖项”，求问怎么解决

![image](https://github.com/sjtug/SJTUThesis/assets/22966522/c3fba977-5d01-4e78-aeb1-69359431533c)

**Reply by LogCreative** [reply]
你可以在原有的环境上更改：

https://github.com/sjtug/SJTUThesis/blob/8e403d73e1f2402e07c21db784d264c5b92b8753/contents/achievements.tex#L3

或者直接修改配置

https://github.com/sjtug/SJTUThesis/blob/8e403d73e1f2402e07c21db784d264c5b92b8753/setup.tex#L102
